test/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+find_package(Boost 1.71 REQUIRED container)
```

Comment:
Missing `COMPONENTS`?

Reply:
Not strictly required but we can add it for the sake of explicitness.